modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/internal/model/listener/test/UserModelListenerTest.java

```diff
@@ -91,6 +105,36 @@ public void tearDown() throws Exception {
 		_companyLocalService.deleteCompany(_company.getCompanyId());
 	}
 
+	@Test
+	public void testOnAfterRemove() throws Exception {
+		_auditMessageProcessor = new TestAuditMessageProcessor();
```

Comment:
What's the purpose of that guy? I didn't get it

Reply:
This TestAuditMessageProcessor is part of configuration needed to run tests with elements there are audited. I've found this config on UserLocalServiceTest.